iphone/Maps/UI/Editor/MWMEditorViewController.mm

```diff
@@ -972,7 +970,7 @@ - (void)tapOnButtonCell:(UITableViewCell *)cell
       GetFramework().CreateNote(self->m_mapObject, osm::Editor::NoteProblemType::PlaceDoesNotExist,
                                 additional);
       [self goBack];
-      [self showDropDown];
+      [self showChangesSentToast];
```

Comment:
```suggestion
      [self showNotesQueuedToast];
```

1. This alert is related to OSM notes only, right? Let's name it properly to avoid confusion with OSM POI tag edits/changes.
2. The "Your suggested changes have been sent" is confusing, because all edits/changes and notes are physically sent to OSM only when the app goes to the background. I suggest to improve the situation by changing `editor_edits_sent_message` translations in `strings.txt` _in a separate pull request_ to:
```
    en = Your note will be sent to OpenStreetMap
    af = Jou nota sal na OpenStreetMap gestuur word
    ar = سيتم إرسال ملاحظتك إلى OpenStreetMap
    be = Ваша нататка будзе адпраўлена ў OpenStreetMap
    bg = Вашата бележка ще бъде изпратена до OpenStreetMap
    ca = La teva nota s'enviarà a OpenStreetMap
    cs = Vaše poznámka bude odeslána do OpenStreetMap
    da = Din note vil blive sendt til OpenStreetMap
    de = Ihre Notiz wird an OpenStreetMap gesendet
    el = Η σημείωσή σας θα σταλεί στο OpenStreetMap
    es = Tu nota se enviará a OpenStreetMap
    es-MX = Tu nota será enviada a OpenStreetMap
    et = Teie märkus saadetakse OpenStreetMapile
    eu = Zure oharra OpenStreetMap-era bidaliko da
    fa = یادداشت شما به OpenStreetMap ارسال خواهد شد
    fi = Huomautus lähetetään OpenStreetMapille
    fr = Ta note sera envoyée à OpenStreetMap
    he = ההערה שלך תישלח אל OpenStreetMap
    hi = आपका नोट OpenStreetMap पर भेजा जाएगा
    hu = Az Ön megjegyzése elküldésre kerül az OpenStreetMap-nek
    id = Catatan Anda akan dikirim ke OpenStreetMap
    it = La tua nota sarà inviata a OpenStreetMap
    ja = あなたのメモはOpenStreetMapに送られる
    ko = 메모가 오픈스트리트맵으로 전송됩니다
    lt = Jūsų pastaba bus išsiųsta į OpenStreetMap
    mr = तुमची टीप OpenStreetMap वर पाठवली जाईल
    nb = Notatet ditt vil bli sendt til OpenStreetMap
    nl = Je opmerking wordt verzonden naar OpenStreetMap
    pl = Twoja notatka zostanie wysłana do OpenStreetMap
    pt = A tua nota será enviada para o OpenStreetMap
    pt-BR = Sua nota será enviada ao OpenStreetMap
    ro = Nota dvs. va fi trimisă către OpenStreetMap
    ru = Ваша заметка будет отправлена в OpenStreetMap
    sk = Vaša poznámka bude odoslaná do OpenStreetMap
    sv = Din anteckning kommer att skickas till OpenStreetMap
    sw = Dokezo lako litatumwa kwa OpenStreetMap
    th = บันทึกของคุณจะถูกส่งไปยัง OpenStreetMap
    tr = Notunuz OpenStreetMap'e gönderilecektir
    uk = Вашу замітку буде надіслано в OpenStreetMap
    vi = Ghi chú của bạn sẽ được gửi tới OpenStreetMap
    zh-Hans = 您的说明将发送至 OpenStreetMap
    zh-Hant = 您的註釋將發送至 OpenStreetMap
```
Make the first commit with strings.txt change, and the second one with the regenerated strings using tools/unix/generate_localizations.sh

Reply:
1. Yes, as I can see it related only to notes, updated method name and rebased PR.
2. Okay, will open another PR!

Reply:
2. Done in https://github.com/organicmaps/organicmaps/pull/6993